src/pages/my/MyPage.tsx

```
@@ -1,7 +1,256 @@
+    isLoading,
+    error,
+  } = useQuery({
+    queryKey: ["userProfile"],
+    queryFn: getUserProfile,
+    retry: false,
+  });
+
+  // 에러 처리 - 401 에러 시 로그인 페이지로 리다이렉트
+  useEffect(() => {
+    if (error) {
+      const status = (error as any)?.status ?? (error as any)?.response?.status;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)